bin/capsule-certs-generate

```diff
@@ -45,11 +45,20 @@ CONFIG_CONTENT = <<EOF
 EOF
 
 config_file = temp_file('capsule-certs', CONFIG_CONTENT)
+
+# generate our parser cache and add it to the config file
+parser_cache = temp_file('capsule-certs.parser_cache', `kafo-export-params -c #{config_file.path} -f parsercache --no-parser-cache`)
```

Comment:
This is a tough one due to the inlined configuration. We didn't during the scenario work because capsule-certs-generate is unique in that it is not an installer scenario but uses kafo and our modules. So its more like a script than anything. We could:

Option 1:
 a) change this to lay down a config file
 b) have that separate from the bin file and not inlined anymore
 c) generate parser cache by pointing at the config file

Option 2: 
 a) see if there is some way to load the file via a script
 b) pass the contents of the CONFIG_CONTENT constant to kafo-export-params in the Rakefile